internal/pkg/admission/policy-server-ca-secret_test.go

```diff
@@ -55,7 +59,7 @@ func TestFetchOrInitializePolicyServerCARootSecret(t *testing.T) {
 	for _, test := range tests {
 		ttest := test // ensure tt is correctly scoped when used in function literal
 		t.Run(ttest.name, func(t *testing.T) {
-			secret, err := ttest.r.fetchOrInitializePolicyServerCARootSecret(context.Background(), generateCAFunc, pemEncodeCertificateFunc)
+			secret, _, err := ttest.r.FetchOrInitializeRootCASecret(context.Background(), generateCAFunc, pemEncodeCertificateFunc)
```

Comment:
it would be nice to add a test for the returned `initialized` boolean, as the reconciler depends on it.